pydm/tests/data_plugins/test_p4p_plugin_component.py

```diff
@@ -51,6 +51,7 @@ def generate_control_variables(value):
             1,
         ),
         (NTEnum().wrap({"index": 0, "choices": ["YES", "NO", "MAYBE"]}), False, 0, 2),
+        (NTScalar("i").wrap({"value": np.int32(10)}), False, np.int32(10), 1),
```

Comment:
it seems like NTScalar.wrap converts the np.int32 into just a python int and it gets passed into the testcase, which is b4 the testcase can try to test it. which also means the `if` on line 82 is not called, (u can add prints to see this and run pytest with _-s_ arg to see print output in terminal)

if we change line 60 to not require a Value type, by having just
`value_to_send,`
we can actually pass the raw np.int32 into the testcase, but then the testcase fails since `send_new_value` function expects a Value type (which doesn't seem to support having a wrapped np.integer type)

im curious to debug what happens when using this fix with the real pydm screen, and why it doesnt hit this issue? Do you have a simple screen calling a PV using an np.integer?